src/lang/bg-BG.json

```diff
@@ -810,7 +810,7 @@
   "histograms_name":"Histograms",
   "histograms_description":"Percentages indicate win rates for the labeled bin",
   "trends_name":"Trends",
-  "trends_description":"Cumulative average over last 500 games",
+  "trends_description":"Moving average of 20 games over last 480 games",
```

Comment:
You only need to update the en-US.json file.  The rest will be filled in automatically during the localization step.